tracer/tracer.go

```diff
@@ -62,10 +71,25 @@ func WithJaegerPropagator(traceId, baggagePrefix string) Option {
 	}
 }
 
+func WithW3CGenerator() Option {
+	return func(t *WavefrontTracer) {
+		t.generator = NewGeneratorW3C()
+	}
+}
+
+func WithW3CPropagator() Option {
+	return func(t *WavefrontTracer) {
+		// implies W3C Generator
+		t.generator = NewGeneratorW3C()
+		t.textPropagator = NewW3CPropagator()
```

Comment:
Maybe rename this to `NewPropagatorW3C` following the convention from the generators?